modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/individual_controller.rb

```diff
@@ -32,6 +32,8 @@ def validate
         def validate_individual_poa_code!(poa_code)
           return if ::Veteran::Service::Representative.where('? = ANY(poa_codes)', poa_code).any?
 
+          return if %w[067 072].include? poa_code
```

Comment:
@mchristiansonVA  What are your thoughts on putting this into a constant to make it more clear why these codes are used here?  Obviously it works either way, just thinking about readability later on.  Like `accepted_ind_poa_codes =` (or whatever for the name) so it is clear why they are being checked to return here